scripts/python/mps/scrapeLoksabhaMPs.py

```
@@ -0,0 +1,481 @@
+    if myneta_const in sansad_lookup:
+        return sansad_lookup[myneta_const][0]
+    return None
+
+# -----------------------
+# MAIN
+# -----------------------
+def main():
+    driver_local = setup_selenium(headless=SELENIUM_HEADLESS)
+    try:
+        # Load ministers first
+        ministers_map = fetch_council_of_ministers(driver_local)
```

Ruff (0.14.3)

Local variable `ministers_map` is assigned to but never used

Remove assignment to unused variable `ministers_map`

(F841)